lib/providers/services/http_service.dart

```diff
@@ -108,7 +108,39 @@ class HttpService {
 
       return members;
     } else {
-      print('Cannot fetch members...');
+      print('Cannot fetch members...'+res.body);
```

Comment:
This debug print shoul be removed.

Reply:
I like to leave in debug statements for error cases - IDK - seems to make sense, it's pretty helpful in development and it's an error condition

That being said would be better to have a real error logging system where we can upload error reports.. these kind of internet failure or server failure errors are good to collect to gauge the user experience.